framework/resources/Functional/schema_change_empty_batch/hbase/emptyHbaseScan1.fail

```diff
@@ -1 +1,3 @@
+alter session set `planner.slice_target`=1;
 select * from schema_change_empty_batch_part t order by t.data.p_partkey limit 5;
+alter session set `planner.slice_target`=100000;
```

Comment:
Why this file still with .fail extension? The associated bug is fixed, right? And now the test should pass? Also use set/reset.

Reply:
Addressed comments.

There were two JIRS's associated with this test case - DRILL-5743, DRILL-5706. Only DRILL-5706 is fixed

Reply:
Can you change to reset? Setting slice_target to 100,000 doesn't make much sense.